### PR TITLE
skip emitting to port for skip default-handling-config

### PR DESCRIPTION
**Pull request by sanjanasandeep**
**Issue**

https://cdap.atlassian.net/browse/PLUGIN-1024

**Description**

skip emitting to port for "skip" default handling config in the router plugin.

### CODE REVIEW

File: src/main/java/io/cdap/plugin/switchcase/route/Router.java [hunk 1]
```diff
@@ -93,7 +93,10 @@ public void transform(StructuredRecord input, MultiOutputEmitter<StructuredRecor
...
         // the value does not match any rule in the port specification, emit it as a
         // defaulting value, based on the selected default handling option
-        emitDefaultValue(input, emitter);
+        if (Config.DefaultHandling.SKIP == emitDefaultValue(input, emitter)) {
+          // handle "skip" default port handling : do not emit to a port 
+           return;
+        }
       } else {
         emitNullValue(input, emitter);
       }
```
Comment (seanzhougoogle) [inline src/main/java/io/cdap/plugin/switchcase/route/Router.java:99]: I think adding a return after this line should be enough for the whole PR.

Comment (sanjanasandeep) [reply src/main/java/io/cdap/plugin/switchcase/route/Router.java:99]: yes, this works. I modified this in the next rev.

File: src/main/java/io/cdap/plugin/switchcase/route/Router.java [hunk 2]
```diff
@@ -97,6 +97,8 @@ public void transform(StructuredRecord input, MultiOutputEmitter<StructuredRecor
...
+      // this is to handle skip default port handling
```
Comment (seanzhougoogle) [inline src/main/java/io/cdap/plugin/switchcase/route/Router.java]: this comment is not correct, it' s not for skipping default port handling.  default value and null value was already handled.

**Review by seanzhougoogle — APPROVED** [review]
LGTM with one minor comment.

File: src/main/java/io/cdap/plugin/switchcase/route/Router.java (hunk 2 above)
Comment (seanzhougoogle) [reply src/main/java/io/cdap/plugin/switchcase/route/Router.java]: I think you don' t need to add any comment here, `return` here should be obvious logic.

Comment (sanjanasandeep) [reply src/main/java/io/cdap/plugin/switchcase/route/Router.java]: removed the comment

**Pull request merged**